COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,96 @@
+# jade Collaborator Guide
```

Comment:
I wonder if we should move the collaborator guide somewhere else entirely (perhaps a page on the website) and then just point new collaborators at this guide, since it's not relevant to most contributors.